crates/sui-bridge/src/types.rs

```diff
@@ -467,6 +467,30 @@ pub struct EthLog {
     pub log: Log,
 }
 
+/// Check if the bridge route is valid
+/// Only mainnet can bridge to mainnet, other than that we do not care.
+pub fn is_route_valid(one: BridgeChainId, other: BridgeChainId) -> bool {
+    if one.is_sui_chain() && other.is_sui_chain() {
+        return false;
+    }
+    if !one.is_sui_chain() && !other.is_sui_chain() {
+        return false;
+    }
+    if one == BridgeChainId::EthMainnet {
+        return other == BridgeChainId::SuiMainnet;
+    }
+    if one == BridgeChainId::SuiMainnet {
+        return other == BridgeChainId::EthMainnet;
+    }
+    if other == BridgeChainId::EthMainnet {
+        return one == BridgeChainId::SuiMainnet;
+    }
+    if other == BridgeChainId::SuiMainnet {
+        return one == BridgeChainId::EthMainnet;
+    }
+    true
```

Comment:
I guess I answered my question above here, why default to true?
what other cases are there that would be fine otherwise (besides the one explicit above?)

Reply:
the cases considered here:
1. transfers between same types of chains (eth <-> eth, sui <-> sui) should be prohibited
2. only mainnet can transfer to mainnet

other than that, everything is allowed